lib/tasks/bower-install.js

```diff
@@ -3,68 +3,97 @@
+      var packages       = options.packages || [];
+      var installOptions = options.installOptions || { save: true };
 
-    ui.startProgress(chalk.green(startMessage));
+      var startMessage = this.formatStartMessage(packages);
+      var completeMessage = this.formatCompleteMessage(packages);
 
-    var config = bowerConfig.read();
-    config.interactive = true;
+      ui.startProgress(chalk.green(startMessage));
 
-    return new Promise(function(resolve, reject) {
-      bower.commands.install(packages, installOptions, config) // Packages, options, config
-        .on('log', logBowerMessage)
-        .on('prompt', ui.prompt.bind(ui))
-        .on('error', reject)
-        .on('end', resolve);
-    })
-      .finally(function() { ui.stopProgress(); })
-      .then(function() {
-        ui.writeLine(chalk.green(completeMessage));
-      });
+      var config = bowerConfig.read();
+      config.interactive = true;
 
-    function logBowerMessage(message) {
-      if (message.level === 'conflict') {
-        // e.g.
-        //   conflict Unable to find suitable version for ember-data
-        //     1) ember-data 1.0.0-beta.6
-        //     2) ember-data ~1.0.0-beta.7
-        ui.writeLine('  ' + chalk.red('conflict') + ' ' + message.message);
-        message.data.picks.forEach(function(pick, index) {
-          ui.writeLine('    ' + chalk.green((index + 1) + ')') + ' ' +
-                       message.data.name + ' ' + pick.endpoint.target);
+      return new Promise(function(resolve, reject) {
+        bower.commands.install(packages, installOptions, config) // Packages, options, config
+          .on('log', logBowerMessage)
+          .on('prompt', ui.prompt.bind(ui))
+          .on('error', reject)
+          .on('end', resolve);
+      })
+        .finally(function() { ui.stopProgress(); })
+        .then(function() {
+          ui.writeLine(chalk.green(completeMessage));
         });
-      } else if (message.level === 'info' && options.verbose) {
-        // e.g.
-        //   cached git://example.com/some-package.git#1.0.0
-        ui.writeLine('  ' + chalk.green(message.id) + ' ' + message.message);
+
+      function logBowerMessage(message) {
+        if (message.level === 'conflict') {
+          // e.g.
+          //   conflict Unable to find suitable version for ember-data
+          //     1) ember-data 1.0.0-beta.6
+          //     2) ember-data ~1.0.0-beta.7
+          ui.writeLine('  ' + chalk.red('conflict') + ' ' + message.message);
+          message.data.picks.forEach(function(pick, index) {
+            ui.writeLine('    ' + chalk.green((index + 1) + ')') + ' ' +
+              message.data.name + ' ' + pick.endpoint.target);
+          });
+        } else if (message.level === 'info' && options.verbose) {
+          // e.g.
+          //   cached git://example.com/some-package.git#1.0.0
+          ui.writeLine('  ' + chalk.green(message.id) + ' ' + message.message);
+        }
       }
-    }
+    }.bind(this));
```

Comment:
yeah, I'll use that in the future. wanted to keep this compatible with the other stuff for now.